ReleaseNotes/RN180814.md

```diff
@@ -0,0 +1,43 @@
+## 180814 (AzSK v.3.5.*)
+  
+### Feature updates
+
+Security Verification Tests (SVTs):
+* New controls for Azure Container Registry (ACR).
+
+ARM Template Checker:
+* The ARM checker now supports multiple API versions for resource types. The versions supported are configurable at the org level.
+
+CICD SVT Extension:
+* Support for adding multiple SVT task instances in the same pipeline. Each instance will generate logs in its own separate folder.
+* SVT task will not ‘silently’ pass if, for some reason, no AzSK scan is performed.
+* Control statuses other than 'Passed' or 'Error' will be now treated as 'Failed' by default.
+* A new pipeline variable 'TreatAsPassed' has been introduced to allow users more control over behavior of the SVT extension in case of various control statuses other than ‘Passed’ or ‘Failed’. For e.g., using this, one may choose to have the extension treat statuses such as 'Verify','Manual','Exception' or 'Remediate' as 'Passed'.
+
+Org-policy feature updates (non-CSE):
```

Comment:
We can remove word "Non-CSE" here. In this context all above changes applies to everyone.